src/components/FolderPage.js

```diff
@@ -0,0 +1,58 @@
+import { useState, useEffect } from 'react';
+import { getFolder } from '../api';
+import CardList from './CardList';
+import search from '../assets/search.svg';
+import './FolderPage.css';
+import linkImg from '../assets/link-Img.png';
+
+
+
+function FolderPage() {
+  const [folderLinks, setFolderLinks] = useState([]);
+
+  const handleLoad = async () => {
+    const folderData = await getFolder();
```

Comment:
```suggestion
    const { folder } = await getFolder();
    const folderLinks = folder?.links;
    
    setFolderLinks(folderLinks);
```

이런 식으로 destruction하는건 어때요?
항상 내부 프로퍼티는 안전하게 접근해주세요 ㅎㅎ